### DISCUSSION TITLE
ZenBusiness Integration

### DISCUSSION THREAD

**Discussion by soulmedicinas** [emailjs-com/help#135] [Q&A]
hey devs,

i have been trying to code in HTML and Javascript to integrate a form in a ZenBusines pop-up window to collect data and send it to the website's email address. the email is powered by Microsoft Outlook. i am using EmailJS to push the colected data to the Outlook address.

i can code for the pop-up but then it will not capture and send the form data. i have used AI to debug the code 10+ times without success. i am losing my mind.

has anyone has success with this? my plan is to also contact a ZB rep as well. it looks like some of the code is being deleted even after saving and republishing, which probably has something to do with the website template parameters.

**Comment by xr0master** [comment]
Unfortunately, we are not familiar with the ZenBusines platform, and it is difficult for us even to guess what your difficulty is.
I can only add that if you find it difficult to use our SDK, you can always use our REST API.

Please try to give more information about what is not working for you and what is working.

**Reply by soulmedicinas** [reply]
hello, thank you for responding.

i can code an external (pop-up) form using HTML. but when i attempt to use Javascript to code the form to submit to an external javascript email service (which is what i was told to use), it does not work. it worked once and then stopped. i'm not sure what changed in the code to prevent it from being functional after only a few attempts. maybe i am not entirely sure how EmailJS works either.

would it help to post the javascript code?

also, this is not necessarily a Git issue. i wrote the HTML code here but pasted it into the ZenBusiness platform.
